### ISSUE TITLE
`root` does not work as intended on windows systems

### ISSUE THREAD

**Issue by DillonSadofsky** [ractivejs/rollup-plugin-ractive-bin#1]
I was noticing if I set the `root` option at all, nothing would be output, no matter what value I enter (as a relative path).  I added a breakpoint to index.js of the module and ran rollup in debug mode and I found the issue was on line 14:
![image](https://user-images.githubusercontent.com/2467337/125114305-70205d80-e0af-11eb-8266-862b01359d8b.png)

startsWith assumes the id will be relative I'm sure, but (at least on windows) its absolute.  I don't really want to put an absolute path into my rollup config, as it'd be deployment specific, and I'm guessing its intended to be relative.  I believe the rollup plugin system has a 'turn absolute id to relative id' method that you could just call before.

**Comment by DillonSadofsky** [comment]
I guess I should say, when I ran with an absolute path in my rollup.config.js, it worked.  So:
`root: 'C:\\Work\\projects\\presage-web\\client'`
but not `root: 'client'`

**Comment by DillonSadofsky** [comment]
I since worked around this.  Its possible it had to do with some translation another rollup plugin was doing or something.  I was using @rollup/plugin-commonjs and rollup-plugin-typescript2.  I've since ditched typescript2 for @rollup/plugin-typescript, swapped the relative `require` statements for fully qualified ES module syntax import statements and it seems to work as designed now.  There very well might still be a case here where it should still work if someone is using a setup like that, but I've since migrated and it seems to work now without modification.

So I probably misread the original issue as it being win32 related, but probably the way that @rollup/plugin-commonjs resolves a relative require path or something.  There is probably still an improvement to be made here, but I guess its working for me now and I've been in bundler hell so long I don't really have it in me to make a reasonable reproduction environment, so I'll close this.
